Sources/EdgeNetworkHandlers/EdgeHitProcessor.swift

```diff
@@ -172,6 +174,9 @@ class EdgeHitProcessor: HitProcessing {
             return nil
         }
 
+        let edgeEndpointStr = configSharedState[EdgeConstants.SharedState.Configuration.EDGE_ENVIRONMENT] as? String
+        let edgeEndpoint = EdgeEndpoint(optionalRawValue: edgeEndpointStr?.lowercased())
```

Comment:
can we move the lowercased inside EdgeEndpoint.init too?

Reply:
I don't mind lowercasing what we read from configuration, but I don't believe the actual enum should be case insensitive.

Reply:
The main reason being is I don't think an enum case should match many different values, rather only the explicit string it defines.

The following behavior I'd also find undesired:
```swift
let rawValue = "INT"
let endpoint = EdgeEndpoint(optionalRawValue: rawValue)

let equal = endpoint.rawValue == rawValue // false
```